ApplicationLibCode/ProjectDataModel/Faults/RimFaultReactivationDataAccessorTemperature.cpp

```diff
@@ -75,8 +77,14 @@ void RimFaultReactivationDataAccessorTemperature::updateResultAccessor()
     m_resultAccessor =
         RigResultAccessorFactory::createFromResultAddress( m_caseData, 0, RiaDefines::PorosityModelType::MATRIX_MODEL, m_timeStep, resVarAddress );
 
-    if ( m_resultAccessor.notNull() )
+    if ( m_resultAccessor0.isNull() )
     {
+        m_resultAccessor0 = RigResultAccessorFactory::createFromResultAddress( m_caseData,
```

Comment:
Check if m_resultAccessor0 is valid after RigResultAccessorFactory::createFromResultAddress(

Reply:
Fixed.